examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>
+		Wearable UI
+	</title>
+	<link href="../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../lib/tau/wearable/theme/default/tau.circle.css"/>
+	<link href="../../css/style.css" rel="stylesheet" />
+	<link href="../../css/style.circle.css"/>
+	<script src="../../js/circle-helper.js">
+	</script>
+	<script src="../../lib/tau/wearable/js/tau.js">
+	</script>
+</head>
+<body>
+    <div class="ui-page">
+        <header class="ui-header">
+            <h2 class="ui-title">Radio</h2>
+        </header>
+        <div class="ui-content">
+            <ul class="ui-listview">
+                <li class="li-has-radio">
+                    <label>Radio on <input type="radio" checked="checked" /></label>
+                </li>
+                <li class="li-has-radio">
+                    <label>Radio off<input type="radio"></label>
```

Comment:
I know that this is example but when I tested this PR all radios are checked.

Reply:
Done!